### PR TITLE
Set CC0 for microaneurysms

### PR DESCRIPTION
**Pull request by ThomasWalter**
## Description

This PR is in answer to issue #3769. 
The CC0 is now clearly stated in the documentation of the load function. 

## Checklist

- [x] [Docstring]

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: skimage/data/__init__.py [hunk 1]
```diff
@@ -178,27 +178,31 @@ def logo():
...
-        """
-        return load("microaneurysms.png")
+    """Gray-level "microaneurysms" image.
+
+    Detail from an image of the retina (green channel).
+    The image is a crop of image 07_dr.JPG from the
+    High-Resolution Fundus (HRF) Image Database:
+    https://www5.cs.fau.de/research/data/fundus-images/
+
+    Notes
```
Comment (hmaarrfk) [inline skimage/data/__init__.py]: I feel like Notes should come after Returns, but such a nit. so who cares.

Comment (hmaarrfk) [reply skimage/data/__init__.py]: Nevermind, seems notes come before returns.

Comment (hmaarrfk) [reply skimage/data/__init__.py]: https://github.com/scikit-image/scikit-image/pull/3625/files

Comment (jni) [reply skimage/data/__init__.py]: @hmaarrfk what made you say that it comes before?

https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Comment (jni) [reply skimage/data/__init__.py]: Ah, why didn't I follow that link? 🤦‍♂️ Anyway, possibly we should change this, but I agree that it should be a separate PR.

**Pull request merged**